orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/pipeline/persistence/ExecutionMapper.kt

```diff
@@ -132,4 +139,31 @@ class ExecutionMapper(
           }
       )
   }
+
+  private fun convertPipelineRefTrigger(execution: PipelineExecution, context: DSLContext) {
+    val trigger = execution.trigger
+    if (pipelineRefEnabled && trigger is PipelineRefTrigger) {
+      val parentExecution = context
+        .select(listOf(
+          field("id"),
+          field("body"),
+          field(name("partition"))
+        ))
+        .from(ExecutionType.PIPELINE.tableName)
+        .where(field("id").eq(trigger.parentExecutionId))
+        .fetchExecutions(mapper, 200, compressionProperties, context, pipelineRefEnabled)
```

Comment:
What's 200?

Reply:
it is the stageReadSize. I set it to 200 because it is the same value we have in `SqlExecutionRepository` but now that you mentioned it, it is not clear what is that 200 use for. I will name the parameter to give more context.